src/service_config/api_config.py

```diff
@@ -25,6 +25,9 @@ def configure_api(service, db_password: str, keypair_paths, hostname, is_prod: b
     generate_api_config_file(service, config_path, db_password, hostname,
                              is_prod, orderly_web_api_url, celery_flower_host)
 
+    print("- Making burden estimate files accessible to task queue")
+    service.api.exec_run("chmod a+w tmp/burden_estimate_files")
```

Comment:
How come this is needed?

Reply:
I think because the task queue worker runs as a specified user called 'worker'

Reply:
Ah, ok. It's probably not a big deal but this does violate the principle of least privilege, I guess a better solution would be to change the latter to run as root?